benchmarks/perf-tool/sample-configs/lucene-hnsw/index.json

```diff
@@ -0,0 +1,27 @@
+{
+  "settings": {
+    "index": {
+      "knn": true,
+      "number_of_shards": 24,
+      "number_of_replicas": 1,
```

Comment:
I see this is different from config for nmslib,  num of replicas and `ef_construction` are different https://github.com/opensearch-project/k-NN/blob/main/benchmarks/perf-tool/sample-configs/nmslib-sift-hnsw/index-spec.json. Do we need to keep all configs identical or it's done this way on purpose?

Reply:
These configs are picked from the S3 bucket, which we were using to do the perf tests. I didn't make any changes.